addons/website_sale/static/src/js/components/media_dialog/website_sale_media_dialog.js

```diff
@@ -0,0 +1,38 @@
+import { MediaDialog, TABS } from "@web_editor/components/media_dialog/media_dialog";
+
+export class WebsiteSaleMediaDialog extends MediaDialog {
+    
+    setup() {
+        super.setup();
+        this.addTab(TABS.VIDEOS);
+    }
+    
+    /**
+     * @override
+     */
+
+    async save() {
+        if (this.errorMessages[this.state.activeTab]) {
+            this.notificationService.add(this.errorMessages[this.state.activeTab], {
+                type: 'danger',
+            });
+            return;
+        }   
+        if(this.state.activeTab == "IMAGES") {
+            const attachments = this.selectedMedia[this.state.activeTab];
+            const preloadedAttachments = attachments.filter(attachment => attachment.res_model);
+            this.selectedMedia[this.state.activeTab] = attachments.filter(attachment => !attachment.res_model);
```

Comment:
```suggestion
            this.selectedMedia[this.state.activeTab] = attachments - preloadedAttachments
```

something like this. code is not tested

Reply:
Adapted the code accordingly to filter out array more accurately